conanfile.py

```diff
@@ -1,12 +1,12 @@
-from conans import ConanFile
-
+from conans import ConanFile, CMake
 
 class FunctionalPlusConan(ConanFile):
     name = "functionalplus"
+    version = "v0.2.8-p0"
     license = "Boost Software License 1.0"
     url = "https://github.com/Dobiasd/FunctionalPlus"
     description = "Functional Programming Library for C++. Write concise and readable C++ code."
-    exports_sources = ["include*", "LICENSE"]
+    exports_sources = ["*", "LICENSE"]
```

Comment:
The project needs to be installed for the `cmake` files to be generated, therefore, all files are to the `conan` build directory. Although, not all may be necessary.

Reply:
I really don't get this change

Reply:
Hi @Dobiasd  and @danimtb , I have modified `export_sources` to:
```cpp
exports_sources = ["examples*", "cmake*", "include*", "CMakeLists.txt", "LICENSE"]
```
If  I leave just `exports_sources = ["include*", "LICENSE"]`, the compilation fails.